syfertext/string_store.py

```diff
@@ -35,7 +37,8 @@ def __contains__(self, string):
 
         return string in self.str_to_key.keys()
 
-    def add(self, string: str):
+    # Some issue with add method
+    def add(self, string:str) -> int:
```

Comment:
is it related to type hints?

Reply:
Yeah, it has to do with typehints, I'll remove that